### ISSUE TITLE
DANGER: 1.13.1 is published to npm and does not appear to reflect your latest source

### ISSUE THREAD

**Issue by lancecaraccioli** [sasstools/sass-lint#1293]
Installing `sass-lint` resulted in issues with airbnb config. I traced it to the version of eslint in the installed package which was 2.x, which led me to your source on github where I see you are using 4.x. I noticed version 1.13.1 is published to NPM however your source shows the package version at 1.13.0. Something is wrong, very wrong.  Here is the npm info for your package from my perspective today.

```bash
[Fri Aug 09 12:34:47] ~/work$ npm info sass-lint

sass-lint@1.13.1 | MIT | deps: 14 | versions: 39
All Node Sass linter!
https://github.com/sasstools/sass-lint

keywords: Sass, SCSS, lint

bin: sass-lint

dist
.tarball: https://registry.npmjs.org/sass-lint/-/sass-lint-1.13.1.tgz
.shasum: 5fd2b2792e9215272335eb0f0dc607f61e8acc8f
.integrity: sha512-DSyah8/MyjzW2BWYmQWekYEKir44BpLqrCFsgs9iaWiVTcwZfwXHF586hh3D1n+/9ihUNMfd8iHAyb9KkGgs7Q==
.unpackedSize: 289.1 kB

dependencies:
commander: ^2.8.1            front-matter: 2.1.2          glob: ^7.0.0                 gonzales-pe-sl: ^4.2.3       known-css-properties: ^0.3.0 lodash.kebabcase: ^4.0.0     path-is-absolute: ^1.0.0     
eslint: ^2.7.0               fs-extra: ^3.0.1             globule: ^1.0.0              js-yaml: ^3.5.4              lodash.capitalize: ^4.1.0    merge: ^1.2.0                util: ^0.10.3                

maintainers:
- bgriffith <gt11687@gmail.com>
- danpurdy <danjpurdy@gmail.com>
- snugug <sam@snug.ug>

dist-tags:
latest: 1.13.1  

published 3 months ago by danpurdy <danjpurdy@gmail.com>
```

**Comment by DanPurdy** [comment]
You're looking at the wrong branch. We release from master branch but our default branch for GitHub is develop. This project has pretty much been unmaintained for over 2 years now but recently and completely accidentally a maintainer pushed out some changes that were scheduled for 2.0..

Basically the develop branch is unsafe to use if anyone clones this as it's in a broken state. Master @1.13.1 is exactly the same almost as v1.12.0

**Comment by lancecaraccioli** [comment]
After looking further I see that you intentionally released 1.13.1. This is still very confusing though. Version 1.13.0 works for us, but 1.13.1 does not work due to the eslint 2.x dependency not working with the latest airbnb configs. Closing ticket

**Comment by DanPurdy** [comment]
https://github.com/sasstools/sass-lint/releases

This should explain a bit further, 1.13.0 may work in some capacity but it should have been a major release at minimum as it includes breaking changes to formatters, node versions, importing configs etc.. it's also unfinished. Unfortunately I never finished it and don't currently have the time to jump back on to continue work on it.

Npm also prevented us from unpublishing a version so we had to revert
